scripts/test_ekf.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+import rospy
+import numpy as np
+import matplotlib.pyplot as plt
+from geometry_msgs.msg import PoseWithCovarianceStamped
+from nav_msgs.msg import Odometry
+from mrover.msg import Waypoint, WaypointType
+from smach_msgs.msg import SmachContainerStatus
+import message_filters
+from util.course_publish_helpers import convert_waypoint_to_gps, publish_waypoints
+from util.SE3 import SE3
+
+
+class EKF_Test:
```

Comment:
General: Even for a test file, it would be a good idea to add doc string comments to the functions, just for future help. If you think it's unnecessary, completely understand.

Reply:
no thats fair, will do